### PR TITLE
[Feature] add OpenZeppelin ERC721 extensions support

### PR DESCRIPTION
**Pull request by alejozarate**
## TL;DR
**A folder structure proposal featuring the addition of ERC721URIStorage.sol support.**

## Problem
It is not possible to inherit from both ERC4907Extension.sol and any OpenZeppelin's ERC721 extension due to ERC721.sol duplicated injection.

## Solution
Create multiple ERC4907Extension with each extension inheriting from the appropriate ERC721 extension.
In order to add support for new extensions, a new contract should be added under `/contracts/ERC4907/extensions/` and modify the corresponding ERC721 injection, as well as adding `/extensions/DelegableTokenExtension[ERC721Extension].sol`.

### CODE REVIEW

**Review by chescalante — APPROVED** [review]
LGTM! Nice job!
